File: backend/src/main/java/com/nbe2_3_3_team4/backend/domain/parking/repository/ParkingRepository.java
package com.nbe2_3_3_team4.backend.domain.parking.repository;

import com.nbe2_3_3_team4.backend.domain.parking.entity.Parking;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ParkingRepository extends JpaRepository<Parking, Long> {

	Parking findByName(String parkingName);

}

File: backend/src/main/java/com/nbe2_3_3_team4/backend/domain/parking/repository/ParkingRepository.kt
package com.nbe2_3_3_team4.backend.domain.parking.repository

import com.nbe2_3_3_team4.backend.domain.parking.entity.Parking
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ParkingRepository : JpaRepository<Parking?, Long?> {
    fun findByName(parkingName: String?): Parking?
}
